backend/de.metas.contracts/src/main/java/de/metas/contacts/invoice/interim/invoicecandidatehandler/FlatrateTermInterimInvoice_Handler.java

```diff
@@ -198,6 +198,12 @@ private I_C_Invoice_Candidate createBaseIC(final @NonNull I_C_Flatrate_Term term
 
 		final I_C_Invoice_Candidate invoiceCandidate = HandlerTools.createIcAndSetCommonFields(term);
 
+		// Currently relying on C_Flatrate_Conditions.InvoiceRule instead of setting it programmatically.
```

Comment:
relying on C_Flatrate_Conditions.InvoiceRule sounds plausible, but *where* are we relyinging on it and in what fashion?
why don't we rely on it by settings it to the IC in here?

Reply:
The InvoiceRule is already set on the IC from the conditions in `de.metas.contracts.invoicecandidate.FlatrateTerm_Handler#createCandidateForTerm`. Will rephrase the comment to make that clearer.